account_statement_import_online_gocardless/models/online_bank_statement_provider.py

```diff
@@ -312,8 +312,11 @@ def _gocardless_obtain_statement_data(self, date_since, date_until):
                     "payment_ref": tr.get(
                         "remittanceInformationUnstructured", partner_name
                     ),
-                    "unique_import_id": tr.get("entryReference", False)
-                    or tr.get("transactionId", False),
+                    "unique_import_id": (
+                        tr.get("entryReference")
+                        or tr.get("transactionId")
+                        or tr.get("internalTransactionId")
```

Comment:
Is it ok to fallback to `None`?

Reply:
Yes, `None` is the same as `False` at ORM level. But any of the 3 should have a value. That's why we add this third one, as if no ID, each download, the transactions will be duplicated.